react/components/ProductSummaryPrice/ProductSummaryPrice.js

```diff
@@ -34,6 +44,38 @@ const ProductSummaryPrice = ({
     sellingPriceClass: 'dib ph2 t-body t-heading-5-ns',
   }
 
+  const sortPrices = (priceA, priceB) => priceA - priceB
```

Comment:
Suggestion: [_sortWith_](https://ramdajs.com/docs/#sortWith)

Reply:
`SortWith`, order by a prop, but we have a list of numbers.